docs/core/tools/dotnet-new.md

```diff
@@ -311,6 +311,8 @@ Each project template may have additional options available. The core templates
 
 `--no-restore` - Doesn't execute an implicit restore during project creation.
 
+`--no-https` - Project doesn't require HTTPS. `app.UseHsts` and `app.UseHttpsRedirection` aren't added to `Startup.Configure`. This option only applies if IndividualAuth or OrganizationalAuth aren't being used.
```

Comment:
On second thought, let's not use the verbiage from the CLI help. It's misleading, since `IndividualAuth` and `OrganizationalAuth` are invalid values, and the usage of either one results in an error. I'll open an issue for this in the CLI repo. How about the following? "This option only applies if `Individual`, `IndividualB2C`, `SingleOrg`, or `MultiOrg` aren't being used."

Reply:
I like that @scottaddie! I also thought that it was confusing to have those values in there if they don't match the real values.

Reply:
Issue can be found here: https://github.com/dotnet/templating/issues/1636

Reply:
I tested them all but windows. Since those values were meaningless I assumed they meant all of auth. It's all but windows.